### ISSUE TITLE
Freezes Upon Loading

### ISSUE THREAD

**Issue by erfinb** [Automatic/tripviewer#5]
When attempting to access the test site from any browser, the page just freezes.  Is this something you could fix, as I love and use this to mark my trips quickly for business use.  Thank you!!! :D

**Comment by brendannee** [comment]
Does it freeze after logging in, or does it freeze even before you log in?

**Comment by erfinb** [comment]
I'm sorry for the delayed response, I didn't get a notification that you had replied.  It happens after I have logged in.  Usually the initial screen loads, then you cannot click on any of the tabs.  If you are able to get into one of the tabs, it then freezes.

**Comment by erfinb** [comment]
Any update on this :)?

**Comment by maroux** [comment]
Hi @erfinb 

Thanks for reporting the issue. If this is still happening, could you post the logs from Firebug Console? You're likely running into an error with one of your trips from which tripviewer isn't able to recover cleanly.

Thanks

**Comment by erfinb** [comment]
Hey!

Is there any way I can do this with Safari? Using developer mode on Safari, and looking at the JavaScript I get this error:

Failed to load resource: the server responded with a status of 503 (Service Unavailable)


Ethan

**Comment by maroux** [comment]
Yeah that should work as well, do you have the request url for that one?

**Comment by baliles** [comment]
I am getting the same behavior. Recreated both my Automatic API app and did a fresh Heroku Button Deploy...made sure all ID/Secrets etc. were correct and updated. When trying to authorize after sign-in using my Automatic Pro account (which works), I get a continuous redirect trying to Authorize the app's scope. Log looks like: 

2017-02-03T16:50:34.822278+00:00 heroku[router]: at=info method=GET path="/redirect" host=demo-vehicle-viewer.herokuapp.com request_id=971b995c-5beb-4421-a0ce-ec326ffba244 fwd="73.137.55.221" dyno=web.1 connect=1ms service=4ms status=302 bytes=340
2017-02-03T16:50:34.790675+00:00 app[web.1]: [0mGET /redirect?state=&code=bb777770ec102c5d62fd5ed40287c9d99c918ca9 [36m302 [0m3.854 ms - 144[0m
2017-02-03T16:50:34.829780+00:00 app[web.1]: [0mGET /redirect [36m302 [0m1.770 ms - 0[0m
2017-02-03T16:50:48.796983+00:00 heroku[router]: at=info method=GET path="/redirect?code=416d02cf7533c1d356d683ab3de0700bdf25848f&state=" host=demo-vehicle-viewer.herokuapp.com request_id=7177596c-b302-449f-a5a1-dace5a4440e5 fwd="73.137.55.221" dyno=web.1 connect=1ms service=37ms status=302 bytes=382
2017-02-03T16:50:48.837511+00:00 heroku[router]: at=info method=GET path="/redirect" host=demo-vehicle-viewer.herokuapp.com request_id=38697c1f-a545-40b3-b9a1-aaef76b82b3c fwd="73.137.55.221" dyno=web.1 connect=4ms service=5ms status=302 bytes=340
2017-02-03T16:50:48.802459+00:00 app[web.1]: [0mGET /redirect?code=416d02cf7533c1d356d683ab3de0700bdf25848f&state= [36m302 [0m1.307 ms - 144[0m
2017-02-03T16:50:48.844420+00:00 app[web.1]: [0mGET /redirect [36m302 [0m2.156 ms - 0[0m
2017-02-03T16:50:50.729743+00:00 heroku[router]: at=info method=GET path="/redirect?code=0b51dac601ec7b07e2ae1fc743741bd6be830ad1&state=" host=demo-vehicle-viewer.herokuapp.com request_id=28e96296-2793-40f3-bd99-b22f236cb148 fwd="73.137.55.221" dyno=web.1 connect=2ms service=14ms status=302 bytes=382
2017-02-03T16:50:50.764706+00:00 heroku[router]: at=info method=GET path="/redirect" host=demo-vehicle-viewer.herokuapp.com request_id=2b55a7ba-ae5c-4483-8689-1caf3122a31a fwd="73.137.55.221" dyno=web.1 connect=1ms service=4ms status=302 bytes=340
2017-02-03T16:50:50.735917+00:00 app[web.1]: [0mGET /redirect?code=0b51dac601ec7b07e2ae1fc743741bd6be830ad1&state= [36m302 [0m3.873 ms - 144[0m
2017-02-03T16:50:50.771277+00:00 app[web.1]: [0mGET /redirect [36m302 [0m1.438 ms - 0[0m

I CAN hit the demo app URL and sign in, but it gives me the Hang instead of the redirect trying to Authorize. Screen shot using the Github Repo's demo URL at https://tripviewer.herokuapp.com that freezes...same as my deployed app was doing, can be seen here: [http://bit.ly/2l5eldh](http://bit.ly/2l5eldh)
